pages/common/ugrep.md

```diff
@@ -0,0 +1,37 @@
+# ugrep
+
+> Ultra fast search tool with query UI, fuzzy search and more.
+> Try also `ugrep --help {{something}}`.
+> More information: <https://github.com/Genivia/ugrep>.
+
+- Start a [q]uery TUI to search files in the working directory recursively (CTRL-Z for help):
+
+`ugrep -Q`
+
+- Search the working directory recursively for files containing a regex pattern:
+
+`ugrep "{{pattern}}"`
+
+- Search for a case-[i]nsensitive regex pattern:
+
+`ugrep -i "{{pattern}}"`
+
+- [L]ist the matching files:
+
+`ugrep -l "{{pattern}}"`
```

Comment:
```suggestion
`ugrep --files-with-matches "{{pattern}}"`
```

Reply:
See earlier comment. Same applies here. The `ag` tool tldr pages also use short options.